packages/components/src/components/brand/brand.lite.tsx

```diff
@@ -86,11 +86,11 @@ export default function DBBrand(props: DBBrandProps) {
 					/>
 				</Show>
 				<Show when={props.anchorChildren}>
-					<>{props.children}</>
+					<strong class="db-brand-title">{props.children}</strong>
 				</Show>
 			</a>
 			<Show when={!props.anchorChildren}>
-				<>{props.children}</>
+				<strong class="db-brand-title">{props.children}</strong>
```

Comment:
```suggestion
				<>{props.children}</>
```

Let's do this with CSS even only.